src/test/java/uk/gov/hmcts/reform/prl/controllers/TestingSupportControllerTest.java

```diff
@@ -153,6 +153,28 @@ public void testExeptionForSubmittedRespondentTaskList() {
         }, RuntimeException.class, "Invalid Client");
     }
 
+    @Test
+    public void testAboutToSubmitCaseCreationCourtNav() throws Exception {
+        when(authorisationService.isAuthorized(any(),any())).thenReturn(true);
+        testingSupportController.aboutToSubmitCaseCreationCourtNav(authToken, s2sToken, callbackRequest);
+        verify(testingSupportService, times(1)).initiateCaseCreationForCourtNav(Mockito.anyString(), Mockito.any(CallbackRequest.class));
+    }
+
+    @Test
+    public void testExeptionForaboutToSubmitCaseCreationCourtNav() {
+        Mockito.when(authorisationService.isAuthorized(authToken, s2sToken)).thenReturn(false);
+        assertExpectedException(() -> {
+            testingSupportController.aboutToSubmitCaseCreationCourtNav(authToken, s2sToken, callbackRequest);
+        }, RuntimeException.class, "Invalid Client");
+    }
```

Comment:
move this file within a package testingSupport similar to main folder

Reply:
Resolved